fstar-helpers/core-models/src/helpers.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(test)]
+pub mod test {
+    use crate::abstractions::{bit::Bit, bitvec::BitVec, funarr::FunArray};
+    use rand::prelude::*;
+
+    /// Helper trait to generate random values
+    pub trait HasRandom {
```

Comment:
`HasRandom` is a question. So I would expect this to tell me if a type has randomness. But this generates randomness.